packages/ckeditor5-dev-env/tests/release-tools/utils/transform-commit/transform-commit-utils.js

```diff
@@ -41,46 +41,170 @@ describe( 'dev-env/release-tools/utils/transform-commit', () => {
 			} );
 		} );
 
-		describe( 'linkGithubUsers()', () => {
-			it( 'marks profile as a link to GitHub profile', () => {
-				expect( transformCommit.linkGithubUsers( '@CKSource' ) )
+		describe( 'linkToGithubUser()', () => {
```

Comment:
Just two tests? Be more creative and also more realistic – these tests check cases completely detached from reality. Those strings will be used in many contexts.